helm/cluster/templates/clusterapi/_helpers_flatcar.tpl

```diff
@@ -42,10 +52,18 @@
     {{- end }}
     {{- if .contents.service }}
     [Service]
+    {{- if .contents.service.environment }}
```

Comment:
You could also repeat this, right? Like, `environment` could be an array?

Reply:
yes that's a good point - i'll switch that over